src/udev/udevd.c

```diff
@@ -1675,12 +1675,16 @@ int main(int argc, char *argv[]) {
 
         if (arg_children_max == 0) {
                 cpu_set_t cpu_set;
+                unsigned long mem_limit;
```

Comment:
It would have been slightly nicer to use uint64_t because physical_memory() returns that type…

Reply:
With this in mind, we should also consider a static upper bound. I don't think more than 10000 workers are reasonable on any system. Actually, the reasonable limit is probably much lower than that.

Reply:
I wasn't worried about overflow, it's mostly about consistency. But it's not very important in this case.